### ISSUE TITLE
Corrupted Datasets

### ISSUE THREAD

**Issue by lts427** [ai4ce/EgoPAT3D#4]
Wonderful work! Recently, I found that some data files of the datasets on the Googledrive have been corrupted, which cannot be unziped. I would appreciate it if you can update the pointcloud data. Thank you!

**Comment by simbaforrest** [comment]
Thank you for raising this issue. @RoboticsYimingLi and I will look into this matter and resolve it soon.

**Comment by lts427** [comment]
> Thank you for raising this issue. @RoboticsYimingLi and I will look into this matter and resolve it soon.

Thank you for your reply. I would appreciate it if you can check the pointcloud data files of the cvpr2022 benchmark posted by this link:
https://drive.google.com/drive/folders/1Eszuzqg0mnGX8fwUjC0YUZbPqtP5xJYp

**Comment by RoboticsYimingLi** [comment]
We have updated the files. Please re-check the files, and contact us again if there is still problems. Thanks!

**Comment by RoboticsYimingLi** [comment]
@lts427

**Comment by simbaforrest** [comment]
Hi @lts427 if the problem is addressed and you are able to use the dataset successfully, could you please close this issue? If not, please let us know what other problems you are experiencing. Thank you.

**Comment by lts427** [comment]
I am sorry that I found some files cannot be decoded normally, e.g, bathroomCabinet. In addition, I am not sure whether you updated all the files, because I can only see that the three folders have been updated (i.e., microwave, stoveTop, and woodenTable). Thank you.

**Comment by ziangcao0312** [comment]
We indeed unzip the files successfully. Please try these two steps:
1. wget --load-cookies /tmp/cookies.txt "https://drive.google.com/uc?export=download&confirm=$(wget --quiet --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate 'https://drive.google.com/uc?export=download&id=16UdiSdBl1Xx9yIpQ98k-VXAcL7Ed1zg6' -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1\n/p')&id=16UdiSdBl1Xx9yIpQ98k-VXAcL7Ed1zg6" -O bathroomCabinet.zip && rm -rf /tmp/cookies. txt
2. unzip basthroomCabinet.zip

**Comment by lts427** [comment]
Thank you for your reply. But the above instructions cannot work (bad request error).
Altough the dataset files cannot be unziped normally (bad CRC error), the code can run. However, the loss will be nan after several iterations. Could you give some suggestions on this phenomenon?

**Comment by simbaforrest** [comment]
@lts427 can you check if you have downloaded the file correctly? And please give more details: which file specifically was problematic for you? Are you downloading it through VPN? Could it be possible that the data was corrupted during your downloading process? Did you check the downloaded file size?

Because we did not find any problem with those data from our end, as @ziangcao0312 mentioned.

**Comment by ziangcao0312** [comment]
We also didn't find the NAN during the training process. I think maybe there are some problems as follows:
1. the data has errors
2. the learning rate is too large (but we didn't face this situation)
3. download the latest codebase.
